src/components/claiming/WalletSeedBackup.vue

```diff
@@ -48,6 +53,9 @@ export default {
         }
       },
     }),
+    disableCopyMnemonics() {
```

Comment:
Could you rename `disableCopyMnemonics` with `isCopyMnemonicsDisabled` here and in
 `src/components/steps/WalletSeedBackup.vue`
```suggestion
    isCopyMnemonicsDisabled() {
```

Reply:
sure